projects/igniteui-angular/src/lib/core/styles/components/bottom-nav/_bottom-nav-theme.scss

```diff
@@ -119,21 +153,26 @@
         position: relative;
         flex-flow: column nowrap;
         flex: 1;
+        gap: if($variant == 'indigo-design', rem(6px), rem(4px));
```

Comment:
In the Indigo UI Kit the gap for the selected item is 6px but in the idle state is 4px.